cmd/simulator/config/keyring.go

```diff
@@ -54,14 +54,14 @@ func NewOffchainKeyring(encryptionMaterial, signingMaterial io.Reader) (*Offchai
 }
```

Comment:
Do we need to have a copy of the keyring or can we use the original one that is currently resides in `chainlink` repo? (and it was done just because of circular dependency)

Reply:
I wanted to avoid pulling anything from `chainlink` because the dependency list grows a lot. The method for signing should not be important. All we need is to apply the interface.